src/main/java/com/exadel/controller/FileUpDownLoadController.java

```
@@ -93,9 +96,9 @@ public FileLoadStatus handleFileDownload(@RequestParam String id, HttpServletReq
                 if (null != outStream)
                     outStream.close();
             } catch (IOException e) {
-                return FileLoadStatus.DOWNLOAD_CLOSING_FAIL;
+                //return FileLoadStatus.DOWNLOAD_CLOSING_FAIL;
```

do not ignore exceptions!